### PR TITLE
feat(composition-helpers): delete `makeProxy` and `makeOptionalProxy`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [ ] Tested with console(if usages are changed) 

### Description
- delete `makeProxy` and `makeOptionalProxy`
- replace `makeProxy`, `makeOptionalProxy` with `useProxyValue`

### CODE REVIEW

File: src/navigation/toolbox/PToolbox.vue [hunk 1]
```diff
@@ -213,10 +212,10 @@ export default defineComponent<ToolboxProps>({
         const initPageSize = props.pageSizeOptions ? props.pageSizeOptions[0] || 24 : 24;
 
         const proxyState = reactive({
-            thisPage: makeOptionalProxy<number>('thisPage', vm, 1),
-            pageSize: makeOptionalProxy<number>('pageSize', vm, initPageSize),
+            thisPage: useProxyValue<number>('thisPage', vm, 1),
+            pageSize: useProxyValue<number>('pageSize', vm, initPageSize),
             sortBy: useProxyValue('sortBy', props, emit),
-            searchText: makeOptionalProxy<string>('searchText', vm, ''),
+            searchText: useProxyValue<string>('searchText', vm, ''),
```
Comment (piggggggggy) [inline src/navigation/toolbox/PToolbox.vue]: Suggestion: `vm` remains

**Review by piggggggggy — APPROVED** [review]
L G T M

File: src/navigation/toolbox/PToolbox.vue (hunk 1 above)
Comment (yuda110) [reply src/navigation/toolbox/PToolbox.vue]: Thanks!!!!!

**Pull request merged**